src/ext/java/org/opentripplanner/ext/flex/trip/ScheduledDeviatedTrip.java

```diff
@@ -43,6 +55,70 @@ public ScheduledDeviatedTrip(Trip trip, List<StopTime> stopTimes) {
     }
   }
 
+  @Override
+  public Stream<FlexAccessTemplate> getFlexAccessTemplates(
+      StopAtDistance access, int differenceFromStartOfTime, ServiceDate serviceDate, DistanceCalculator calculator
+  ) {
+    DistanceCalculator scheduledDistanceCalculator = new ScheduledDistanceCalculator(calculator, this);
+
+    int fromIndex = getFromIndex(access);
+
+    if (fromIndex == -1) return Stream.empty();
+
+    ArrayList<FlexAccessTemplate> res = new ArrayList<>();
+
+    for (int toIndex = fromIndex + 1; toIndex < stopTimes.length; toIndex++) {
+      if (stopTimes[toIndex].dropOffType == 1) continue;
```

Comment:
Could you use the constant here. The ultimate goal would be to have enums in the internal model.